bitnami/mongodb/templates/NOTES.txt

```diff
@@ -187,7 +187,7 @@ To connect to your database from outside the cluster execute the following comma
 
 To access the MongoDB&reg; Prometheus metrics, get the MongoDB&reg; Prometheus URL by running:
 
-    kubectl port-forward --namespace {{ .Release.Namespace }} svc/{{ include "mongodb.fullname" . }}-metrics {{ .Values.metrics.service.ports.metrics }}:{{ .Values.metrics.service.ports.metrics }} &
+    kubectl port-forward --namespace {{ .Release.Namespace }} svc/{{ include "mongodb.service.nameOverride" . }}-metrics {{ .Values.metrics.service.ports.metrics }}:{{ .Values.metrics.service.ports.metrics }} &
```

Comment:
[metrics-svc.yml](https://github.com/bitnami/charts/blob/main/bitnami/mongodb/templates/metrics-svc.yaml#L5) uses  `mongodb.fullname` template for name.

Reply:
I didn't notice the `-metrics`, nice catch!